### DISCUSSION TITLE
Customs Hints

### DISCUSSION THREAD

**Discussion by UndertaleUHC** [GitbookIO/community#404] [Feature Requests]
Hello there!

As a passionate enthusiast of hints, I believe it would greatly enhance the user experience if you could introduce a feature to create custom hints. Currently, we have informative hints (i), successful hints (v), warning hints (!), and danger hints /!\. However, envisioning a new button that empowers users to craft hints with customizable colors for the left bar, symbols, and their corresponding shapes would be a fantastic addition. This feature not only promises to be useful but also seems quite feasible to implement. Let users add their creative touch to hints, making the interaction more personalized and engaging. Looking forward to the evolution of this idea!

Here's an example of how this feature could be visually represented (created quickly using Paint.NET):
![image_2023-11-26_135027933](https://github.com/GitbookIO/community/assets/150550865/ac2060ba-88bb-4e96-b300-d6b59586f0b8)

**Comment by katco-6** [comment]
Yes please!! I came here specifically to request this. The ability to have no icon, choose your own icon, and choose your own color, ideally. We'd like to use them for "Recommended" and "Required" elements of our design and language system. The current hint types and symbols don't really match those.
